docs/README.md

```diff
@@ -1,60 +1,93 @@
+
+To start a local Handsontable documentation server:
+
+1. From the `handsontable/docs` directory, install the documentation dependencies:
+    ```bash
+    npm install
+    ```
+2. From the `handsontable/docs` directory, generate the API reference:
+   ```bash
+   npm run docs:api
+   ```   
+3. From the `handsontable/docs` directory, start your local documentation server:
+   ```bash
+   npm run docs:start
+   ```
+4. In your browser, go to: http://localhost:8080/docs/.
+
+## Documentation npm scripts:
+
+From the `handsontable/docs` directory, you can run the following npm scripts:
+
+* `npm run docs:start` – Starts a local documentation server at `localhost:8080/docs/`.
+* `npm run docs:start:no-cache` – Starts a local documentation server without cache.
+* `npm run docs:api` – Generates the Handsontable API reference into `/next/api`.
+* `npm run docs:build` – Builds the documentation output into `/.vuepress/dist` and checks for broken links.
+* `npm run docs:build:no-check-links` – Builds the documentation output into `/.vuepress/dist`.
+* `npm run docs:docker:build` – Builds a Docker image for the staging environment.
+* `npm run docs:docker:build:staging` – Builds a Docker image for the staging environment.
+* `npm run docs:docker:build:production` – Builds a Docker image for the production environment.
+* `npm run docs:version <semver.version>` – Creates a new documentation version in a new `/<semver.version>/` directory.
+* `npm run docs:check-links` – Checks for broken links.
+* `npm run docs:lint` – Runs ESLint on the `/next/` directory's content.
+* `npm run docs:lint:fix` – Runs ESLint on the `/next/` directory's content and auto-fixes problems.
+* `npm run docs:assets:next` – Prepares the `next` documentation version's CSS and JavaScript.
+
+## Handsontable documentation directory structure
+
+```bash
+docs                            # All documentation files
+├── .vuepress                   # All VuePress files
+│   ├── components              # Vue components
+│   ├── containers              # Markdown containers
+│   │   ├── examples            # Code examples container
+│   │   └── sourceCodeLink.js   # `source-code-link` container.
+│   ├── handsontable-manager    # A module that runs Handsontable examples in different Handsontable versions
+│   ├── public                  # The documentation's public (static) assets
+│   ├── theme                   # Theme overwrites and customizations
+│   ├── tools                   # Our custom documentation tools
+│   │   ├── check-links.js      # The documentation's link checker
+│   │   ├── jsdoc-convert       # JSDoc-to-Markdown converter
+│   │   ├── utils.js            # Tools utilities
+│   │   └── version             # A tool that creates new documentation versions
+│   ├── config.js               # VuePress configuration
+│   ├── docs-links.js           # Lets us link to the documentation's latest version with `@` (e.g. [link](@/guides/path/file.md.)
```

Comment:
It does not link to the `latest`. it links in the scope of the currently selected version.

For instance:

* `[link](@/api/metaSchema.md#autorowsize)` in `/next/` generate link:`[link](/docs/next/api/options#autorowsize)`
* `[link](@/api/metaSchema.md#autorowsize)` in `/9.0/` (latest) generate link:`[link](/docs/api/options#autorowsize)`
* `[link](@/api/metaSchema.md#autorowsize)` in `/8.3/` generate link:`[link](/docs/8.3/api/options#autorowsize)`